test/everypolitician/popolo/membership_test.rb

```diff
@@ -51,4 +51,9 @@ def test_membership_end_date_method_always_present
   def test_membership_person
     assert_equal 'Bob', popolo.memberships.first.person.name
   end
+
+  def test_membership_equality
+    memberships = Everypolitician::Popolo.read('test/fixtures/estonia-ep-popolo-v1.0.json').memberships.to_a
+    assert_equal false, memberships[0] == memberships[1]
```

Comment:
I think `refute_equal memberships[0], memberships[1]` is cleaner and more explicit, and would give better diagnostics in the case of failure.